### PR TITLE
First release notes

### PR DESCRIPTION
**Pull request by rr1000**
### Changes
- Added release notes section
- Added first release notes for what I assume is 1.0 - from this doc https://docs.google.com/document/d/1F97J7P5U0xhAcSpBcZdAcOhCDRs8eT0tnswDQCywJXs/edit?pli=1#

### What's needed
Before publishing we'll need to fill in the proper dates for when we plan to do the release.

### CODE REVIEW

File: source/sections/release-notes/22-release-notes-2018-01-16.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- The deployments for ingress, coredns, elasticsearch, and prometheus-operator are now configured to land on controller nodes, leaving more capacity on worker nodes for customer workloads.
+- System volumes and snapshots will have tags for faster recovery in the event of a disaster.
+
+#### Bug Fixes
+
+- Fixed internal Prometheus communication issues enabling grafana to pull in more default data.
+
+#### Customer Support:
+
+- In order to tend to your support issue in a timely manner, please submit your ticket through the Platform dashboard by clicking on the “Contact Support” button located in the footer of the Environment UI. This provides valuable metadata to the support staff, which allow them to triage the issue much quicker.
```
Comment (forana) [inline source/sections/release-notes/22-release-notes-2018-01-16.md]: should include a link to the product dashboard here

File: source/sections/release-notes/22-release-notes-2018-01-16.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Our Support Policies: Support is provided in English from our offices in Madison, WI.
+- Support hours are Monday through Friday 9:00 a.m. to 5:00 p.m. CST.
```
Comment (forana) [inline source/sections/release-notes/22-release-notes-2018-01-16.md]: should we just link to policies instead of duplicating them here?

**Comment by forana** [comment]
LGTM

File: source/sections/release-notes/22-release-notes-2018-01-16.md (hunk 2 above)
Comment (anthonypleshek) [reply source/sections/release-notes/22-release-notes-2018-01-16.md]: Removed for now

File: source/sections/release-notes/22-release-notes-2018-01-16.md (hunk 1 above)
Comment (anthonypleshek) [reply source/sections/release-notes/22-release-notes-2018-01-16.md]: Done.

**Pull request merged**